Code/GraphMol/MolDraw2D/DrawMol.h

```diff
@@ -0,0 +1,301 @@
+//
+//  Copyright (C) 2021-2022 David Cosgrove and other RDKit contributors
+//
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+// Original author: David Cosgrove (CozChemIx Limited)
+//
+// This class is a helper class used by MolDraw2D to draw an ROMol.
+// It is not part of the public API and is not intended to be used except
+// by MolDraw2D.
+
+#ifndef RDKIT_DRAWMOL_H
+#define RDKIT_DRAWMOL_H
+
+#include <map>
+#include <string>
+#include <vector>
+
+#include <Geometry/point.h>
+#include <GraphMol/MolDraw2D/AtomSymbol.h>
+#include <GraphMol/MolDraw2D/DrawAnnotation.h>
+#include <GraphMol/MolDraw2D/DrawShape.h>
+#include <GraphMol/MolDraw2D/MolDraw2DHelpers.h>
+
+namespace RDKit {
+
+class Atom;
+class Bond;
+class ROMol;
+class RWMol;
+class DrawText;
+
+namespace MolDraw2D_detail {
+
+class DrawMol {
+ public:
+  virtual ~DrawMol() = default;
+
+  // Make the object, scaled to a given pixel size.
+  /*!
+    \param mol             : the molecule to draw
+    \param legend          : the legend (to be drawn under the molecule)
+    \param width           : width (in pixels) of the rendering
+    set this to -1 to have the canvas size set automatically
+    \param height          : height (in pixels) of the rendering
+    set this to -1 to have the canvas size set automatically
+    \param highlight_atoms : (optional) vector of atom ids to highlight
```

Comment:
This and the other multi-word parameters are all mis-named. Should be highlightAtoms